catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveTable.java

```diff
@@ -132,8 +135,22 @@ public static HiveTable fromHiveTable(Table table) {
             table.getPartitionKeys().stream()
                 .map(p -> identity(p.getName()))
                 .toArray(Transform[]::new))
-        .withSchemaName(table.getDbName())
-        .build();
+        .withSchemaName(table.getDbName());
+  }
+
+  public CachedClientPool clientPool() {
+    return clientPool;
+  }
+
+  public void close() {
+    if (clientPool != null) {
+      clientPool.close();
+      clientPool = null;
+    }
+  }
```

Comment:
Do you need to close here, since it will also be used in `CatalogOperations`?

Reply:
fixed and add comment